src/lib/components/input/input.js

```diff
@@ -1,83 +1,77 @@
 
-class Input extends PureComponent {
-  static propTypes = {
-    ...componentPropTypes,
-    placeholder: PropTypes.string,
-    disabled: PropTypes.bool,
-    required: PropTypes.bool,
-    name: PropTypes.string.isRequired,
-    value: PropTypes.string,
-    type: PropTypes.string,
-    autoFocus: PropTypes.bool,
-    onChangeFunction: PropTypes.func,
-    // props from withValidation HOC
-    renderValidationError: PropTypes.func,
-    handleValidations: PropTypes.func.isRequired,
-    isValidationOk: PropTypes.func.isRequired,
-  };
+const Input = ({
+  theme,
+  name,
+  value,
+  error,
+  label,
+  className,
+  type,
+  onChangeFunction,
+  handleValidations,
+  helperText,
+  containerClassName,
+  ...inputProps
+}) => {
+  const handleChange = event =>
+    handleInputChange(event, name, handleValidations, onChangeFunction);
 
-  static defaultProps = {
-    ...defaultComponentPropTypes,
-    placeholder: '',
-    disabled: false,
-    required: true,
-    value: '',
-    type: 'text',
-    onChangeFunction: () => {},
-  };
-
-  handleChange = name => event =>
-    handleInputChange(event, name, this.props.handleValidations, this.props.onChangeFunction);
-
-  render() {
-    const {
-      containerClassName,
-      className,
-      theme,
-      placeholder,
-      disabled,
-      required,
-      name,
-      value,
-      type,
-      isValidationOk,
-      renderValidationError,
-      autoFocus,
-      // removing props that are not valid for DOM elements
-      // to hand the rest to the input
-      handleValidations,
-      onChangeFunction,
-      getValidationMessage,
-      validationMessage,
-      hasErrorAfterSubmit,
-      // props handed to the input DOM element
-      ...restProps
-    } = this.props;
-
-    return (
-      <div className={`${containerClassName} ${styles[`theme-${theme}`]} ${styles.inputWrapper}`}>
+  return (
+    <div className={styles[`theme-${theme}`]}>
+      <div className={styles.wrapper}>
         <input
-          placeholder={placeholder}
-          className={`${isValidationOk() && styles.error} ${styles.input} ${className}`}
-          value={value}
+          className={`${className} ${error && styles.error}`}
+          type={type || "text"}
+          id={name}
           name={name}
-          type={type}
-          required={required}
-          onChange={this.handleChange(name)}
-          disabled={disabled ? 'disabled' : ''}
-          autoFocus={autoFocus}
-          {...restProps}
+          onChange={handleChange}
+          value={value}
+          {...inputProps}
         />
-        {renderValidationError()}
+        <label htmlFor={name}>
+          <span className={styles.fieldName}>{label}</span>
+        </label>
+        <span className={`${styles.errorMessage} ${error && styles.active}`}>
+          {error}
+        </span>
+        {!error && helperText && (
+          <span className={styles.helperMessage}>{helperText}</span>
+        )}
```

Comment:
@rin why are you changing the input component in this PR too that is not related. Be careful because you are adding again the error with the validation that is solved in Staging